OpenGLUnitTests/Matrix4Tests.cs

```diff
@@ -76,14 +78,28 @@ public void MatrixSubtract()
         [TestMethod]
         public void MatrixMultiply()
         {
-            Matrix4 m1 = new Matrix4(new double[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 });
-            Matrix4 m2 = new Matrix4(new double[] { 10, 20, 30, 40, 50, 10, 20, 30, 40, 50, 10, 20, 30, 40, 50, 10 });
-            Matrix4 mult = m1 * m2;
-
-            Assert.AreEqual(mult[0], new Vector4(220, 230, 190, 100));
-            Assert.AreEqual(mult[1], new Vector4(740, 710, 630, 500));
-            Assert.AreEqual(mult[2], new Vector4(1260, 1190, 1070, 900));
-            Assert.AreEqual(mult[3], new Vector4(1780, 1670, 1510, 1300));
+            Matrix4 m1 = new Matrix4(new double[] {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15});
+            Matrix4 m2 = new Matrix4(new double[] {10, 20, 30, 40, 50, 10, 20, 30, 40, 50, 10, 20, 30, 40, 50, 10});
+            Matrix4 mult = m2 * m1;
+
+            Assert.AreEqual(new Vector4(220, 230, 190, 100), mult[0]);
+            Assert.AreEqual(new Vector4(740, 710, 630, 500), mult[1]);
+            Assert.AreEqual(new Vector4(1260, 1190, 1070, 900), mult[2]);
+            Assert.AreEqual(new Vector4(1780, 1670, 1510, 1300), mult[3]);
+        }
+        
+        [TestMethod]
+        public void MatrixMultiplyMultipleMatrix()
+        {
+            Matrix4 m1 = new Matrix4(new double[] {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15});
```

Comment:
My suggestion above would apply here as well, where I think the formatting of these arrays should match existing formatting.